sdk/parents/azure-code-customization-parent/pom.xml

```diff
@@ -0,0 +1,60 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../azure-client-sdk-parent/pom.xml</relativePath>
+  </parent>
```

Comment:
Do we need to inherit from client parent pom?

Reply:
Hmm, probably not but it's nice to have the parent POM handle all plugin version upgrades,